src/sota_tools/ostree_http_repo_test.cc

```diff
@@ -3,7 +3,9 @@
 #include "deploy.h"
 #include "ostree_http_repo.h"
 #include "ostree_ref.h"
+#include "server_credentials.h"
```

Comment:
Do we really need this lnclude?

Reply:
Probably yes, because I don't see it included in `ostree_http_repo.h`.